stl/inc/thread

```diff
@@ -179,6 +179,24 @@ _NODISCARD auto _To_absolute_time(const chrono::duration<_Rep, _Period>& _Rel_ti
     return _Abs_time;
 }
 
+struct _Clamped_rel_time_ms_count_result {
+    unsigned long _Count;
+    bool _Clamped;
+};
+
+template <class _Duration>
+_NODISCARD _Clamped_rel_time_ms_count_result _Clamped_rel_time_ms_count(const _Duration& _Rel) {
+    // _Clamp must be less than 2^32 - 1 (INFINITE) milliseconds, but is otherwise arbitrary.
+    constexpr chrono::milliseconds _Clamp{chrono::hours{24}};
+
+    if (_Rel >= _Clamp) {
+        return {static_cast<unsigned long>(_Clamp.count()), true};

```

Comment:
It didn't matter in `this_thread::sleep_until()`, the original source of this code, but now `_Clamped_rel_time_ms_count()` cares about whether clamping happened. We should test for strictly greater-than here - if the user asked for exactly 24h, then we don't need to clamp.

Note that this is what the now-removed `_To_timespec64_sys_10_day_clamped()` did - it said

```cpp
const bool _Clamped      = _Ten_days_d < _Rel_time;
```